lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks if an example includes any expectation.
+      #
+      # All RSpec's example and expectation methods are covered by default.
+      # If you are using your own custom methods,
+      # add the following configuration:
+      #
+      #   RSpec:
+      #     Language:
+      #       Examples:
+      #         Regular:
+      #           - custom_it
+      #       Expectations:
+      #         - custom_expect
+      #
+      # @example
+      #
+      #   # bad
+      #   it do
+      #     a?
+      #   end
+      #
+      #   # good
+      #   it do
+      #     expect(a?).to be_truthy
```

Comment:
That just my two cents. Method whose name ends with `?`, generally return boolean type. So, `be true` would be more suitable as an example than `be_truthy`.

```suggestion
      #     expect(a?).to be(true)
```

Reply:
Yeah, I used to think so too (`a?` should always return true or false), but Rails, for example, is known not to have such policy.

- https://github.com/rails/rails/pull/19424
    - Example of `ActionDispatch::Request#xhr?` returning Integer

Reply:
For https://github.com/rails/rails/pull/19424, the API is already widespread, so maybe a trade-off with breaking changes may have been considered. Proposal that will change return value of published API will be cautious.

Reply:
Yes, that's true, and there's no reason not to include this suggested change. I'll add this change, thanks 👍